### PR TITLE
Also omit square brackets for Pandoc Citation in CAYW

### PR DESCRIPTION
**Pull request by jpcirrus**
Incorporate the change proposed in #358 and implemented in 8b260cb to CAYW as well.

### CODE REVIEW

File: chrome/content/zotero-better-bibtex/cayw.coffee [hunk 1]
```diff
@@ -203,7 +203,7 @@ Zotero.BetterBibTeX.CAYW.Formatter = {
       cite += " #{citation.suffix}" if citation.suffix
       formatted.push(cite)
     return '' if formatted.length == 0
```
Comment (retorquere) [inline chrome/content/zotero-better-bibtex/cayw.coffee]: This line is superfluous if no brackets are returned

**Comment by retorquere** [comment]
Would you mind adjusting the test at https://travis-ci.org/ZotPlus/zotero-better-bibtex/jobs/117784481#L724 to make sure the build goes green? That would allow a clean merge.

**Comment by jpcirrus** [comment]
Sorry about the runaround. I'm not a developer and know nothing about coffescript or javascript. Next time I'll just raise an issue.

**Comment by retorquere** [comment]
No runarounds, I appreciate people putting in the effort; it's just that if I merge as-is, the git history would claim the effort as mine instead of yours.

**Comment by HJAllen** [comment]
A bit late to the party but i am using a workflow of RStudio IDE, rmarkdown, and pandoc. I respectfully disagree with the notion that not having quick copy with brackets is a better. Much like the original requester for removing the brackets, I am submitting that having the brackets is better for me! Is it possible to some way add this as an option?
Thanks!
Joel

**Comment by retorquere** [comment]
Preference added [here](https://github.com/retorquere/zotero-better-bibtex/releases/download/builds/zotero-better-bibtex-1.6.93-br463-3584.xpi). If you could confirm this is what you want, I can merge.

**Comment by HJAllen** [comment]
Perfect! Thanks!

H. Joel Allen, PhD
Environmental Scientist
Office of Research and Development
United States Environmental Protection Agency
(513) 487-2806
allen.joel@epa.gov

**Comment by retorquere** [comment]
Alright, you can use this for now, you will be upgraded automatically somewhere in the next few days (schedule is out of my hands, it's a FF setting), but it will just be what you have now with a bumped version number.

**Comment by retorquere** [comment]
(oh and hang in there -- you're doing important and necessary work regardless of what your president or your new boss may think)

**Pull request merged**